CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 # Chef Server Changelog
 
+## 12.4.0 (2016-01-27)
+
+### Components
+
+#### Updated
+* openssl (1.0.1p -> 1.0.1q)
+* knife-opc (528be923 -> 0b8fa0fa)
+* ohai (f1e35bf1 -> 81f1c968)
+* chef (2fe875ce -> 3f3fbc8f)
+
+#### New
+* rest-client (1.8.0)
+
+#### Removed
+* chef-server-bootstrap
+
+## Detailed Change Log
+
+* `oc-pedant`
+    * Replace /policies/:group/:name in spec descriptions with /policy_groups/:group_name/policies/:policy_name.
+    * fix spec descriptions that were copied from /cookbooks to cookbook_artifacts.
+    * Allow opt-out of RVM/bundler busting in knife pedant tests
+    * Add validation tag to header validation test
+
+* `oc-erchef`
+    * added ACL endpoints for policies and policy groups; also pedant tests
+    * Implement RFC 14 - Add universe endpoint
+    * V1 of Server Admins. Implements flexable user management global group.
+    * Fix statem test output formatting
```

Comment:
the fixed test output was only for failing cases, most people probably wouldn't care about this